darts/models/__init__.py

```diff
@@ -12,6 +12,7 @@
 from .theta import Theta, FourTheta
 from .arima import ARIMA
 from .fft import FFT
+from .transformer_model import TransformerModel
```

Comment:
@inverniz Could you move this import to after line 30 (inside the try/except)? That should make the tests pass.

Reply:
@hrzn I pulled your changes, moved the import line, and updated the TransformerModel tests to align with the most recent RNN tests. Locally, I still have three tests failing that appear to be unrelated to my changes. Please see the attached screenshot. The three tests seem to fail because performance of a few model is below a certain threshold. I am running the tests by launching the coverage.sh script from the terminal.
<img width="944" alt="Schermata 2020-10-08 alle 10 43 58" src="https://user-images.githubusercontent.com/17392130/95450539-b866ef80-0966-11eb-9138-e7cd72d02071.png">

Reply:
After looking at the test logs of the PR, it seems that the failing task is "coverageTest_core". Indeed, I see that the overall coverage is only 77% (I suppose you want 80 or more) because TransformerModel has only 5% test coverage. This is weird because locally I have 87% overall coverage and 100% coverage for TransformerModel (see screenshot). Maybe I still have to import the tests for TransformerModel somewhere? 
<img width="553" alt="Schermata 2020-10-08 alle 13 16 19" src="https://user-images.githubusercontent.com/17392130/95451852-aede8700-0968-11eb-8e71-af9b5bc6cc68.png">

Reply:
Hi @inverniz, what is hitting you might be caused by the fact that we accepted a PR that breaks down the code into models to develop, which might change test/coverage results. Can you make sure you are up-to-date with the develop branch and retrigger the tests?

Sorry for the issues and pls ping me if it doesn't help.

Reply:
Hi @inverniz,

I'm not quite sure yet why the first tests are failing but we'll take a deeper look.

Concerning the coverage tests, it's due to a mistake on our side with the way we run them in the `build.gradle` file. I'm working on fixing this and will report back to you.

Reply:
HI @LeoTafti and @TheMP, thank you for your inputs. I am also looking into the issue. It seems to me that the latest version of the develop branch was successfully merged in my PR (the last commit in the develop branch is 2 days old, @hrzn did the merge yesterday), so that shouldn't be the issue. I will let you know when I find out more.

Reply:
I believe it worked because we got lucky, test coverage when running `./gradelew coverageTest_core` barely reached the minimum 80% if you look at [this run](https://github.com/unit8co/darts/runs/1214649410) and the same behavior observed for the `TransformerModel` can be seen for other files which require dependencies not included in the `core` flavor of the library as well (e.g. line 49 of the same run states `darts/models/rnn_model.py                        44     43     2%   7-172`).

Reply:
I see that #212 has been merged. I suppose I should update the branch and trigger the tests again, correct?

Reply:
Correct! It should work now, ping me if that's not the case